cea/datamanagement/zone_helper.py

```diff
@@ -90,7 +90,11 @@ def clean_attributes(shapefile, buildings_height, buildings_floors, buildings_he
             np.nanmedian(data_floors_sum_with_nan))  # median so we get close to the worse case
         shapefile["floors_ag"] = [int(x) if not np.isnan(x) else data_osm_floors_joined for x in
                                   data_floors_sum_with_nan]
-        shapefile["height_ag"] = shapefile["floors_ag"] * constants.H_F
+        if 'height' in list_of_columns:
+            shapefile["height_ag"] = shapefile['height'].fillna('0')
+            shapefile["height_ag"] = shapefile["height_ag"].where(shapefile["height_ag"]==0,shapefile["floors_ag"] * constants.H_F)
```

Comment:
A couple of things:
1. In line 94, NaNs are filled as a string, but in line 95 the condition is specified as an int. I don't think this is desired behavior, correct?
2. More importantly, this `where` appears to do exactly the opposite of what it's supposed to do: it's filling in all the values that were _not_ NaN as equal to `shapefile["floors_ag"] * constants.H_F`. Or at least that's what happens when I run this code on my computer?
3. I think these two lines can be written in a more foolproof way:
`shapefile["height_ag"] = shapefile['height'].fillna(shapefile["floors_ag"] * constants.H_F)`